agents_json/devopness/agents.json

```diff
@@ -0,0 +1,206 @@
+          "sourceId": "devopness",
+          "operationId": "addEnvironmentServer"
+        }
+      ],
+      "links": [
+        {
+          "origin": {
+            "actionId": "devopness_list_projects",
+            "fieldPath": "responses.200.0.id"
+          },
+          "target": {
+            "actionId": "devopness_get_project",
+            "fieldPath": "parameters.project_id"
+          }
+        },
+        {
+          "origin": {
+            "actionId": "devopness_get_project",
+            "fieldPath": "responses.200.id"
+          },
+          "target": {
+            "actionId": "devopness_list_project_environments",
+            "fieldPath": "parameters.project_id"
+          }
+        },
+        {
+          "origin": {
+            "actionId": "devopness_list_project_environments",
+            "fieldPath": "responses.200.0.id"
+          },
+          "target": {
+            "actionId": "devopness_get_environment",
+            "fieldPath": "parameters.environment_id"
+          }
+        },
+        {
+          "origin": {
+            "actionId": "devopness_get_environment",
+            "fieldPath": "responses.id"
+          },
+          "target": {
+            "actionId": "devopness_list_credentials",
+            "fieldPath": "parameters.environment_id"
+          }
+        },
+        {
+          "origin": {
+            "actionId": "devopness_list_credentials",
+            "fieldPath": "responses.200.0.id"
+          },
+          "target": {
+            "actionId": "devopness_get_credential",
+            "fieldPath": "parameters.credential_id"
+          }
+        },
+        {
+          "origin": {
+            "actionId": "devopness_get_credential",
+            "fieldPath": "responses.id"
+          },
+          "target": {
+            "actionId": "devopness_create_server",
+            "fieldPath": "requestBody.credential_id"
+          }
+        }
+      ],
+      "fields": {
+        "parameters": [
+          {
+            "name": "hostname",
+            "type": "string",
+            "required": true,
+            "description": "The server name and hostname to be set on Linux cloud servers. Accepts numbers (0-9), dash (-) and lower case non accented characters. Must not be greater than 60 characters."
+          },
+          {
+            "name": "provision_input",
+            "required": true,
+            "type": "object",
+            "description": "Configuration used to launch the new server as a virtual machine instance in the cloud provider",
+            "properties": {
+              "credential_id": {
+                "type": "integer",
+                "description": "The unique id of the cloud provider credential to be used on connect to cloud provider",
+                "example": 8430015757
+              },
+              "cloud_service_code": {
+                "$ref": "#/components/schemas/ServerCloudServiceCode"
+              },
+              "settings": {
```

Comment:
@yagnyaPatel 
In the fields block is it okay to reference types defined in OpenAPI spec?
Or do we need to duplicate all attributes of each field with "type": "object"?
See for example the cloud_service_code field below defined in agents.json

```json
"cloud_service_code": {
  "$ref": "#/components/schemas/ServerCloudServiceCode"
},
```
Is this allowed? If not, what's the alternative?